README.md

```diff
@@ -121,6 +121,8 @@ increase web server performance:
 
 ### Development
 
+Set the database access credentials by editing `config/parameters.yml`.
```

Comment:
This should go into the "installation" section, not into the "development" section.

Reply:
Also why not just use environment variables, the cleanest approach and easy also to automate for future CI